src/libinjection_xss.c

```diff
@@ -341,14 +649,20 @@ static attribute_t is_black_attr(const char* s, size_t len)
     }
 
     if (len >= 5) {
-        /* JavaScript on.* */
+
+	/* JavaScript on.* event handlers */
         if ((s[0] == 'o' || s[0] == 'O') && (s[1] == 'n' || s[1] == 'N')) {
```

Comment:
Since we check for `on` here, should we change the array values to not include the `on` prefix for when we compare on line 657?

Reply:
@migueldemoura I've rebased, and started comparing from the third char after removing `on` prefix from the array values. Can you review it again?